html_css/flexbox.md

```diff
@@ -0,0 +1,28 @@
+### Introduction
+
+Floats were never intended for layout, however clever developers found ways in which to use them to create their layouts. Having worked with floats you may have noticed some limitations and bugs (clearfix anyone? verticle centering?).
+
+### Learning Outcomes
+
+* How do you define a flex container?
+* What items will become a flex-item?
+* Can a flex item also be a flex container?
+* What is the difference between `justify-content` and `align-items`.
+* How can you change the direction of the axis of Flexbox. What will this do to `justify-content` and `align items`?
+* How do you change the display order of content with Flexbox. What is a drawback with this (for screen readers)?
+
+### Overview
+
+Flexbox is a new layout mode in CSS3 initially developed by Mozilla to make layouts. There were 3 different iterations of the flex property, the final 2012 spec — `display: flex` has excellent [browser support](https://caniuse.com/#search=flex).  The current Flexbox spec is fully supported in all modern browsers, including mobile, IE11 and Edge.
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+* Watch [Wes Bos'](https://twitter.com/wesbos) tutorial [What the Flexbox?!](https://flexbox.io/). This course consists of 20 videos covering Flexbox and are all available to [watch on Youtube](https://www.youtube.com/playlist?list=PLu8EoSxDXHP7xj_y6NIAhy0wuCd4uVdid). 
```

Comment:
Maybe we don't need to link to Wes' Twitter. We're already using his resources all over anyway, and having the 2 links so close together might disrupt skim readers.